spyder/widgets/variableexplorer/tests/test_dataframeeditor.py

```diff
@@ -174,6 +179,36 @@ def test_change_format_with_format_not_starting_with_percent(qtbot, monkeypatch)
     with qtbot.assertNotEmitted(editor.sig_option_changed):
         editor.change_format()
 
+def test_header_encoding():
+    data =  u"""a,Unieke_Idcode,ü,b,1
+            0,flooooo,0.55,10100103,31/12/2016
+            """
+    test_data = StringIO(data)
```

Comment:
@ccordoba12 I will change the `StringIO` and read a file directly 👍 , however the file that you point has as encoding `utf-8 BOM` (`'utf-8-sig'`) , taking this into account, is this the correct file to use in the encoding test or maybe I need to create another one with an unicode character different from the BOM one?